rllab/misc/tensor_summary.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+import numpy as np
+import tensorflow as tf
+from tensorboard import summary as summary_lib
+from tensorboard.plugins.custom_scalar import layout_pb2
+
+from rllab.misc.console import mkdir_p
+
+
+class Summary:
```

Comment:
1. Classes should always have a name matching the file containing them, e.g. `tensor_summary.py` --> `TensorSummary` or `summary.py` --> `Summary`